### ISSUE TITLE
Potato swaps to itself on new input in Windows 10??

### ISSUE THREAD

**Issue by tmdrae80** [talvo/potato#447]
Hi. I've been a long time user of potato since I was pointed at it. This is a recent issue. It is like the title says. I could be watching something someone has linked me or the like, and the program swaps to itself as soon as new activity happens in any of the windows. I have all the activity check boxes unchecked and I have rechecked them. It's an irritating issue I'm hoping there might be a bit of support for? Please help!

**Comment by fuzzyweapon** [comment]
YES.  This is a huge problem.  Here are some things I've noticed:

- Potato will maximize a minimized instance and bring it to the top of the window stack each time there is new activity (I don't **think** it's actually switching focus to that window like tmdrae80 is reporting)
- Potato does not /reliably/ maximize on new activity (possible bug here since it is intermittent)
- There does not seem to be a way to disable this functionality
  - Functionality is undocumented in the wiki/F1 help.
  - If there **is** a way to disable it, it is likely tied to an existing option which already has some other functionality tied to it, meaning the two options are not implemented orthogonally at the user level.  

Some comments on how this affects usability:
- If I'm at work, I don't want Potato popping up when I can't control it.  This is a deal breaker.
- As tmdrae80 said, it also disrupts other activities you'd want not to be interrupted.
- When features such as this are not implemented orthogonally in terms of options (if this is) at a user level and it's not immediately obvious to the user what all features a particular option enables/disables, it means users have to do lots of **testing** just to use the client.  This greatly increases the barrier to adoption or retention, since every time I want to tweak something, I have to re-learn what everything does, and in this particular case, that involves testing.

Usability praise and feedback!:
I still think this is an awesome feature!  There are lots of times where I can see myself wanting the client window to pop up because, frankly, sometimes you just don't see systray and taskbar notifications.  Also, I might want to switch over to something like some full screen netflix while waiting for things like poses, etc.

Expected:
- have a way to turn this on and off independently from other options
